### DISCUSSION TITLE
indicate, in Web UI, whether timer is enabled by changing color of corresponding tab (1-16)

### DISCUSSION THREAD

**Discussion by discip** [arendst/Tasmota#22682] [Ideas]
Since the option of indicating the status of a switch/button in the Web UI is now available, this could potentially be added as well.

**Have you looked for this feature in other issues and in the docs?**  
_It seems, that this is not issued yet._

**Is your feature request related to a problem? Please describe.**  
_No_

**Describe the solution you'd like**  
_Currently you have to enter the specific timer tab, to see, if it is enabled or not.
Would be much more convenient, if one could see that, without having to enter every single 'time-tab'._

**Describe alternatives you've considered**  
_-_

**Additional context**

This is just an idea of how it could look like. 😊

![suggestion](https://user-images.githubusercontent.com/53649486/95784059-63630a80-0cd3-11eb-815d-77e8cf802b50.png)

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by discip** [comment]
@arendst
Is this difficult to implement, or does it require a significant amount of storage?

**Comment by Jason2866** [comment]
Not Theo but answering, an Idea (feature request) is something that maybe considered or not. If this step has been taken it still needs someone which takes the stab and does implement. There is **no** road map for anything which will be done in Tasmota. Things (features) happens :-)
Since Tasmota is Open Source a PR is welcome!

**Reply by s-hadinger** [reply]
Also a general rule is that important information should not be indicated by color only - there is a significant portion of the population color-blind. You may argue though that this is not important information.
